test/end-to-end/components/Blueprint.component.js

```diff
@@ -26,7 +26,7 @@ class Blueprint {
   }
 
   get blueprintNameList() {
-    const selector = ".list-group-item-heading";
```

Comment:
@henrywang I wanted to make sure you saw this change. I included data attributes `[data-description]` above and `[data-blueprint]` here to use instead of class names for the tests. The new class names in the PF4 component are more generic and not as useful for targeting specific contents on the page. I also like data attributes better than class names, because I know these attributes are applied for the purpose of testing. Whereas when we use class names for testing, it isn't obvious when updating the UI which selectors are used for tests until I break the tests :-)

Reply:
NICE!! Thanks @jgiardino!